src/python/pants/python/python_setup.py

```diff
@@ -120,6 +120,17 @@ def register_options(cls, register):
                 "high, may result in starvation and Out of Memory errors."
             ),
         )
+        register(
+            "--resolver-http-cache-ttl",
+            type=int,
+            default=3_600,  # This matches PEX's default.
```

Comment:
I wasn't sure how to model this. I considered `default=None`, which would mean to use PEX's default value of 3600. But I think that breaks the abstraction and requires users to have intimate knowledge of PEX's default options.

Reply:
Also I didn't want to change from Pex's default here, until we have a better handle on https://github.com/pantsbuild/pex/issues/1084.